ports/raspberrypi/boards/solderparty_bbq20kbd/pins.c

```diff
@@ -0,0 +1,41 @@
+#include "shared-bindings/board/__init__.h"
+
+STATIC const mp_rom_map_elem_t board_module_globals_table[] = {
+    CIRCUITPYTHON_BOARD_DICT_STANDARD_ITEMS
+
+    { MP_ROM_QSTR(MP_QSTR_INT), MP_ROM_PTR(&pin_GPIO0) },
+    { MP_ROM_QSTR(MP_QSTR_PERIPHERAL_SDA), MP_ROM_PTR(&pin_GPIO28) },
+    { MP_ROM_QSTR(MP_QSTR_PERIPHERAL_SCL), MP_ROM_PTR(&pin_GPIO29) },
```

Comment:
In 8.0.0 we're renaming I2C peripheral to target. Want to match it here?

Reply:
Hmm, on the one hand, it would be nice to keep it consistent, but on the other hand, it doesn't feel very clear to name the I2C that lets you use the keyboard as a peripheral "Target I2C". Or is it just me? :sweat_smile:

Reply:
Up to you.

Reply:
Let's keep it as is for now :)

Reply:
`KEYBOARD_I2C`?